Dockerfile-debian.template

```diff
@@ -50,14 +51,16 @@ RUN set -ex; \
     ; \
     \
 # pecl will claim success even if one install fails, so we need to perform each install separately
-    pecl install APCu-%%APCU_VERSION%%; \
-    pecl install memcached-%%MEMCACHED_VERSION%%; \
-    pecl install redis-%%REDIS_VERSION%%; \
+    pecl install APCu-5.1.14; \
+    pecl install memcached-3.0.4; \
+    pecl install redis-4.2.0; \
+    pecl install imagick-%%IMAGICK_VERSION%%; \
```

Comment:
Something went wrong here...

Reply:
What went wrong? When I ran `update.sh`, I noticed it is substituting this variable properly.

Reply:
Never mind I see what you mean. Let me fix that.

Reply:
You replaced the variable with the hardcoded version in the Debian template. The Alpine template is fine.